docs/docs/configuration/message-tracking.md

```diff
@@ -31,31 +31,31 @@ in Frontend, Consumers and Management.
 
 ### Frontend configuration
 
-* create `ElasicsearchClientFactory`, which will produce ElasticSearch driver and store it for cleanup
-* set log repository via `HermesFrontend.Builder#withLogRepository`
-* add shutdown hook to cleanup the driver
```

Comment:
It is not directly related to this line, but the line reminded me that in the consumers module I added the shutdown hook here: https://github.com/allegro/hermes/blob/cf78bb22a936fe9326bc3d00db851d42d1976d0b/hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/config/ConsumerConfiguration.java#L221 Probably we should do the same in the frontend module.